massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -63,10 +42,6 @@ pub struct ConsensusWorker {
     latest_final_periods: Vec<u64>,
     /// clock compensation
     clock_compensation: i64,
-    /// staking keys
-    staking_keys: Map<Address, KeyPair>,
```

Comment:
@damip @Eitu33 Now which module should manage the staking keys? Should I integrate this in PoS?

Reply:
Isnt' this already handled by [BlockFactoryWorker](https://github.com/massalabs/massa/blob/feature/add_pos_and_selector_skel/massa-factory-worker/src/block_factory.rs#L23) `wallet` field? That's what I understood in today's morning presentation.

Reply:
The node wallet containing the staking keys is created in node main and an arc mutex is passed to the api (for key management) and another is passed to factory which uses it for production